rosidl_default_runtime/rosidl_default_runtime-extras.cmake.in

```diff
@@ -22,3 +22,5 @@ foreach(_dep ${rosidl_runtime_packages})
     endif()
   endif()
 endforeach()
+
+include_directories(${@PROJECT_NAME@_INCLUDE_DIRS})
```

Comment:
I don't think any package should set global `include_directories` in its CMake config file? Why was this added?

Reply:
`rosidl_default_generators` is used in every message package and somehow it was including the directories when `rosidl_generator_c/cpp` were not splited. Whitout this line when a message package is been compiled it's not able to find the `rosidl_runtime_c/cpp` headers.

any tip about where I should include this headers?

Reply:
You can reference the exact line where it was happening before the split?

Reply:
For example here: https://github.com/ros2/common_interfaces/blob/master/std_msgs/CMakeLists.txt#L15

A possible sollution could be to include these lines:

```
find_package(rosidl_default_runtime REQUIRED)
include_directories(${rosidl_default_runtime_INCLUDE_DIRS})
```

Reply:
In that case we should close this PR and open some PRs like this https://github.com/ros2/unique_identifier_msgs/pull/6, right?

Reply:
That PR still uses `include_directories`.